components/templates/ArticleTemplate/ArticleTemplateStyles.ts

```diff
@@ -8,8 +8,15 @@ export const StyledHeaderImage = styled.img`
 `
 export const StyledAuthorByline = styled.p`
     color: #6c757d;
+    margin: 0 0 0.5rem;
```

Comment:
I changed the margins a little. The margin for the dates I made based on how they looked on the video posts